nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -277,6 +285,24 @@ public class PutDatabaseRecord extends AbstractProcessor {
             .allowableValues("true", "false")
             .defaultValue("true")
             .build();
+    public static final PropertyDescriptor TRANSLATION_STRATEGY = new PropertyDescriptor.Builder()
+            .required(true)
+            .name("Column Name Translation Strategy")
+            .description("The strategy used to normalize table column name")
```

Comment:
Do all the non-regex options uppercase the name for translation purposes? If so that's fine but somewhere (maybe here) the doc should reflect that no matter the strategy (unless possibly the new Regex one), the name will be uppercased to do case-insensitive matching

Reply:
Hi @mattyb149 
I have updated the documentation